src/collections.js

```diff
@@ -0,0 +1,86 @@
+import {
+  actionToPromise,
+} from 'core';
+
+/** Create an action that runs several actions at once and resolves as soon
+ * as any one of the actions resolves.
+ *
+ * NOTE: once its actions have started, this action is not responsible for
+ * stopping them. For example, if you've started playing several audio
+ * files at once and you want them all to stop when the first one stops,
+ * this action doesn't provide for that. Caveat programmator!
+ *
+ * @param {action} action An action.
```

Comment:
style: Separate the description from the argument name using a dash
```suggestion
 * @param {action} action - An action.
```